Makefile.inc

```diff
@@ -142,7 +142,11 @@ lint: gitlint golangci-lint markdownlint yamllint
 
 # [markdownlint] validates Markdown files in the project
 markdownlint:
-	markdownlint markdownlint -c .markdownlint.yml -i vendor .
+	md_ignored=(vendor); \
+	if [ -r .mdignore ]; then \
+		md_ignored+=($$(< .mdignore)); \
+	fi; \
+	markdownlint -c .markdownlint.yml $${md_ignored[@]/#/-i }
```

Comment:
Oops, sorry,

```suggestion
	markdownlint -c .markdownlint.yml $${md_ignored[@]/#/-i } .
```

Reply:
Gave me a chance to make some fixes to the commit message too. :wink: Thanks @skitt!